lib/src/definitions.dart

```diff
@@ -381,7 +389,7 @@ class MapViewError {
 
   const MapViewError({required this.code, required this.message});
 
-  static MapViewError NoNetworkError() {
+  static MapViewError noNetworkError() {
```

Comment:
Esto usábamolo internamente certo ? Non rompemos a ninguén ?

Reply:
Tecnicamente podemos romper porque é un método público, pero o máis probable é que non. En dart o lint protesta vivamente por estas cousas e quixen cambialo.